src/main/drivers/timer.h

```diff
@@ -85,11 +85,12 @@ typedef struct timerHardware_s {
 #endif
 } timerHardware_t;
 
-enum {
+typedef enum {
+    TIMER_INPUT_ENABLED = 0x00,
     TIMER_OUTPUT_ENABLED = 0x01,
     TIMER_OUTPUT_INVERTED = 0x02,
     TIMER_OUTPUT_N_CHANNEL= 0x04
-};
+} timerFlag_t;
```

Comment:
Although betaflight is not entirely consistent about this I prefer `enum` definitions to end `_e`, so this should be `timerFlag_e`. A purist would disagree, since this is a type definition, but the `_e` suffix provides useful info when coding (for example, `enum`s cannot be forward declared in C).